core/src/main/scala/cromwell/core/WorkflowSourceFiles.scala

```diff
@@ -1,9 +1,28 @@
 package cromwell.core
 
+import better.files.File
 import wdl4s.{WdlJson, WdlSource}
 
 /**
   * Represents the collection of source files that a user submits to run a workflow
   */
-final case class WorkflowSourceFiles(wdlSource: WdlSource, inputsJson: WdlJson,
-                                     workflowOptionsJson: WorkflowOptionsJson)
+
+sealed trait WorkflowSourceFilesCollection {
+  def wdlSource: WdlSource
+  def inputsJson: WdlJson
+  def workflowOptionsJson: WorkflowOptionsJson
+}
+
+final case class WorkflowSourceFiles(wdlSource: WdlSource,
+                                     inputsJson: WdlJson,
+                                     workflowOptionsJson: WorkflowOptionsJson) extends WorkflowSourceFilesCollection {
+
+  def defaultImports: WorkflowSourceFilesWithImports = {
+    WorkflowSourceFilesWithImports(wdlSource, inputsJson, workflowOptionsJson, None)
+  }
+}
+
+final case class WorkflowSourceFilesWithImports(wdlSource: WdlSource,
+                                                inputsJson: WdlJson,
+                                                workflowOptionsJson: WorkflowOptionsJson,
+                                                importsFile: Option[File]) extends WorkflowSourceFilesCollection
```

Comment:
If I understand this change correctly, if we have the `WorkflowSourceFilesWithImports ` type, we should disambiguate based on `WorkflowSourceFilesWithImports ` vs `WorkflowSourceFiles ` rather than `Some` vs `None`. So this doesn't need to be an `Option`

ToL: Personally I wouldn't mind a single `case class` with an `Option`, but if we have both we should use both